components/utils/Toast.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react';
+import Image from 'next/image';
+import styles from './toast.module.css';
+
+interface ToastProps {
+  onClose: () => void;
+  type: 'success' | 'error';
+  message: string;
+}
+
+export const Toast: React.FC<ToastProps> = ({ type, onClose, message }) => {
```

Comment:
Cohérence entre les différents composants : 
```suggestion
export const Toast = ({ type, onClose, message }: ToastProps) => {
```